src/content/configuration/dev-server.md

```diff
@@ -892,7 +892,7 @@ module.exports = {
 Usage via the CLI
 
 ```bash
-npx webpack serve --open-page /different/page1,/different/page2
+npx webpack serve --open-page /different/page1 --open-page /different/page2
```

Comment:
I believe `--open-page` should be used along with `--open`, otherwise it's sort of useless example.

If I pass `/concept`, the browser will open this url `http://localhost:8080//concepts/` instead of `http://localhost:8080/concepts/`. Can you double check it?

Reply:
Ah yes bug in v3,  fixed in beta. 
It should be `npx webpack serve --open --open-page different/page1 --open-page different/page2 ` with a note regarding `//`

Reply:
`--open` saved for compatibility, but it will be removed, future usage:

- Open page
```
webpack --open-target
```
- Open specific page
```
webpack --open-target /index.html
```
- Open multiple specific pages
```
webpack --open-target /index.html webpack --open-target /bar.html
```
- Open firefox:
```
webpack --open-app firefox
```
- Open specific page with frefox
```
webpack --open-target foo.html --open-app firefox
```